src/Surfnet/StepupRa/RaBundle/Security/Authentication/Token/SamlToken.php

```diff
@@ -48,6 +48,8 @@ class SamlToken extends AbstractToken
     private $raManagementInstitution;
 
     /**
+     * The SHO of the identity
+     *
```

Comment:
Question remains, what differentiates this field from the identityInstitution? When I read that description it states:

> The identity institution is set with the SHO of the identity. This value is not overridden like the user institution can be

The newly introduced field seems to do the same.

Reply:
Thanks for clearifying the original use case of the identityInstitution IRL. Because it seemed duplicate we've decided to remove it.